code/modules/mob/living/silicon/posi_brainmob.dm

```diff
@@ -62,7 +62,7 @@
 		return FALSE
 
 	owner_channels.Cut()
-
+/*
```

Comment:
This seems to just get channels from the mob the posi_brain is currently driving so it can state laws into it. Don't think it should be commented-out/require re-write, at least not in this PR.